Sources/Screens/RepositoryDetail/RepositoryDetailView.swift

```diff
@@ -1,11 +1,111 @@
+                }
+                Section("COUNTS") {
+                    HStack(alignment: .firstTextBaseline) {
+                        Text("⭐️")
+                            .countsTextModifier()
+                        Text("stars".capitalized)
+                        Spacer()
+                        Text("\(viewModel.repository.stargazersCount)")
+                    }
+                    HStack {
+                        Text("⫚")
+                            .countsTextModifier()
+                        Text("Forks")
+                        Spacer()
+                        Text("\(viewModel.repository.forks)")
+                    }
+                    HStack {
+                        Text("👀")
+                            .countsTextModifier()
+                        Text("Watchers")
+                        Spacer()
+                        Text("\(viewModel.repository.watchers)")
+                    }
+                }
+                Section("DESCRIPTION") {
+                    Text("Language: \(viewModel.repository.language ?? "n/a")")
+                        .textFontModifier(size: 20, weight: .semibold)
+                    Text(
+                        "Details: \(viewModel.repository.description ?? "n/a")"
+                    )
+                    .font(.title2)
+                }
+
+                Section("DATES") {
+                    Text(
+                        "Created at: \(viewModel.repository.createdAt, style: .date)"
+                    )
+                    .font(.headline)
+                    Text(
+                        "Updated at: \(viewModel.repository.updatedAt ?? Date.now, style: .date)"
+                    )
+                    .font(.headline)
+                }
+
+                Section("OWNER") {
+                    HStack {
+                        Text("\(viewModel.repository.owner.login)")
+                            .textFontModifier(size: 30, weight: .bold)
+                            .foregroundColor(.white)
+                    }
+                    .frame(minHeight: 100)
+                    .background {
+                        ZStack {
+                            AsyncImage(url: URL(
+                                string: viewModel.repository.owner
+                                    .avatarUrl
+                            )) { image in
+                                image
+                                    .resizable()
+                                    .aspectRatio(contentMode: .fill)
+                            } placeholder: {
+                                Color.gray
+                            }
+                            .scaledToFill()
+                            .blur(radius: 10)
+                            Color(.black)
+                                .opacity(0.3)
+                        }
+                        .frame(width: 700, height: 150)
+                    }
+                }
+                .onTapGesture {
```

Comment:
avoid using .onTapGesture, this is why we have Buttons and NavigationLinks etc. You can use Button here instead of adding tap gesture on the HStack itself.